HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -152,7 +152,7 @@ static int32_t HdmiArcPortID = -1;
 
 #define API_VERSION_NUMBER_MAJOR 1
 #define API_VERSION_NUMBER_MINOR 0
-#define API_VERSION_NUMBER_PATCH 12
+#define API_VERSION_NUMBER_PATCH 13
```

Comment:
Pls update patch version to 14 to match changelog

Reply:
updated the API patch version.